Assets/Scripts/Weapons/ProjectileGun/Gun.cs

```diff
@@ -53,6 +53,8 @@ private void Fire()
         float rotation = Mathf.Atan2(relativePoint.y, relativePoint.x) * Mathf.Rad2Deg + 90;
 
         // Plays sound effect.
+        // Test Use AudioManipulation.BitCrusher(Data.SoundEffect, 11);

```

Comment:
Why 11? I don't think I fully understand what is happening here as it does nothing up to 11? I'm confused.

Reply:
A noticeable difference in sound tends to be from halving it so its 2^11 Hz

Reply:
I still don't think I fully understand. If this is fully correct could you perhaps add a comment to explain why the number needs to be `11`. I don't have a problem with it just the behavior is confusing to me.

Reply:
It doesn't need to be eleven. It could be ten or twelve, but the file will follow 2^11 Hz which means the file will compress to that size. When the file becomes more and more compressed it makes the file sound more retro. We can decrease the number to increase the compression because the Hz would be less. The Bitcrusher is using file compression to alter the sound. Its to the power because decreasing a file by say 10% or a set number isn't very noticeable.

Reply:
Ok, can you add a comment that quickly explains that please.